pyproject.toml

```diff
@@ -4,7 +4,21 @@ requires = [
     "packaging",
     "wheel",
     "cython>=0.29.20",
-    "numpy>=1.16.6,<1.20",
+    # See https://numpy.org/doc/stable/user/depending_on_numpy.html for
+    # the recommended way to build against numpy's C API:
+    "oldest-supported-numpy",
     "scipy>=1.0",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.cibuildwheel]
+manylinux-x86_64-image = "manylinux2014"
+manylinux-i686-image = "manylinux2014"
+# Change in future version to "build"
+build-frontend = "pip"
+
+[tool.cibuildwheel.override]
+# NumPy support manylinux2010 on CPython 3.6-3.9
+select = "cp3?-*"
```

Comment:
This will need updating if you want to build 3.10 wheels as well.  Which takes precedence - this, or the CIBW environment variable?

Reply:
The override is only meant to be applied to Python 3.6 to 3.9 for where we've build with manylinux2010 before rather than manylinux2014 for QuTiP 4.6.x. From QuTiP 4.7 onwards we can build with manylinux2014 everywhere.

Reply:
Ah right, I understand thanks.